handlers/services.go

```diff
@@ -13,16 +15,15 @@ import (
 )
 
 const (
-	defaultMetricsRange = "5m"
-	namespaceParam      = "namespace"
-	serviceParam        = "service"
-	rangeParam          = "range"
+	metricsDefaultRateInterval = "1m"
+	metricsDefaultStepSec      = 15
+	metricsDefaultDurationMin  = 30
 )
 
 // ServiceList is the API handler to fetch the list of services in a given namespace
 func ServiceList(w http.ResponseWriter, r *http.Request) {
 	params := mux.Vars(r)
-	namespace := models.Namespace{Name: params[namespaceParam]}
+	namespace := models.Namespace{Name: params["namespace"]}
```

Comment:
Minor comment, but just to follow same guideline.
We normally used constants in Java, and we followed that here, but I see you move the constant to plain string in some cases (not only on this PR), what is the preferred way. I'm not in pro/cons but I want to understand the style criteria.
If no style defined, I would respect the existing one unless there is a motivation for the change.

Of course, this is not blocker.

Reply:
Hmmm ... I introduced these constants at the beginning and then I saw noone was using them, so I decided to follow the others :)

Reply:
It's minor, but I happened also in others PR I reviewed, so I decide to leave a note :-).